scheme/casper/client.go

```
@@ -0,0 +1,220 @@
+		request.Header.Set("Content-Type", "application/json")
+	}
+
+	httpClient := c.httpClient
+	if httpClient == nil {
+		httpClient = http.DefaultClient
+	}
+	response, err := httpClient.Do(request)
+	if err != nil {
+		return err
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)